app/models/user.rb

```diff
@@ -1,10 +1,11 @@
 class User < ApplicationRecord
-  has_secure_password
   has_many :reservations
   has_many :hotels, through: :reservations
+  has_many :room_types, through: :reservations
+
+  has_secure_password
 
-  validates :email, presence: true, uniqueness: true
-  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :email, presence: true
+  validates :password, presence: true
```

Comment:
Well done adding validations @Tadessejemal1 and @Kumilachew-g, however, I think that setting the email validation of uniqueness would be nice, since you don't want many users with the same email, it could mess up everything.